### ISSUE TITLE
Chainsaw while player shooting bug

### ISSUE THREAD

**Issue by blajockz** [LVPlayground/playground#817]
it's a been a while before recording taht bug which is for chainsaw, when player that you wnat to chainsaw him is sniping or afk without using esc, when you start chainsawing him it wont work directly, you have to shoot him with anyother weapon then chainsaw him so it work correctly, iheard its an sampcac bug which is anti cheat and not supposed to make weapons "bugged" this bug is so annoying much ashell

**Comment by FinishaLVP** [comment]
Yes, I'm also aware of this bug. Chainsaw sync can't be fixed, it's a **SA:MP problem** and not SAMPCAC problem, it tends to get desynced and doesn't deal any damage to your opponents even if they're stunned and not able to move. As you said, you'll have to distance yourself from your opponents and simply try again
